tests/integration_tests/test_teammatchmaker.py

```diff
@@ -330,7 +335,12 @@ async def test_game_matchmaking_multiqueue_timeout(lobby_server):
     )
     assert msg["state"] == "stop"
 
-    # Don't send any GPGNet messages so the match times out
+    await client_response(protos[0])
+
+    await asyncio.gather(*[
+        idle_response(proto) for proto in protos[1:2]
+    ])
```

Comment:
isn't this identical to `await idle_response(protos[1])`?

Reply:
Oh good catch. I think it should be `protos[1:3]` but I'll double check